www/source/blog/2018-10-12-shelving-composites.html.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Shelving Composites (for now)
+date: 2018-10-12
+author: Garrett Amini
+tags: supervisor, packaging
+category: supervisor
+classes: body-article
+---
+
+Nearly exactly one year ago today, Christopher Maier [presented the ability to create _composite packages_](https://www.habitat.sh/blog/2017/10/Introducing-Composites/), a special kind of Habitat package that includes other Habitat packages in order to be managed by a single Supervisor. The Habitat community had been finding all sorts of creative ways to use Habitat to incorporate their services under a single Supervisor, so we sought a way to canonically enable them to do so. 
+
+This past year has taught us that while we were on the right track with the idea of composite packages, our implementation left much to be desired. Users kept trying to utilize composite packages, but would inevitably find themselves bumping up against limitations of the feature, and would thus end implementing the same sorts of creative solutions we were trying to simplify in the first place.
+
+## What's Happening Now?
+
+Starting with version *X.XX.X*, we will be removing the ability to run composite plans from Habitat. We've learned enough over the past year to realize that we need a complete re-implementation of how that feature will work with the Supervisor, so we don't want to lead our users down a dead-end path. This is one of those cases where rebuilding the feature will be much faster to implement than refactoring what we have, so we are taking the hard step of removing the current implementation now.
+
+## The Path Forward
+
+As we near 1.0, we are seeking to make Habitat as robust and reliable as possible. Once we achieve that milestone, we're looking forward to once again adding the ability to create composite packages, developed with help and feedback from the community.
+
+If you have feedback that will help us drive this in the right direction, we'd love to hear from you [in the forums](https://forums.habitat.sh). Happy Habitizing!
```

Comment:
It might be a good idea to start a forum post on this, and give the link to that post here (rather than a link to the forums in general), just as a way to centralize feedback.